src/main/java/de/simonscholz/telegram/bot/api/domain/Audio.java

```diff
@@ -0,0 +1,5 @@
+package de.simonscholz.telegram.bot.api.domain;
+
+public class Audio {
```

Comment:
Could you at least also add these annotation to all empty domain objects you created so far?
```
@Data
@JsonIgnoreProperties(ignoreUnknown = true)
```